RxCocoa/iOS/UITextField+Rx.swift

```diff
@@ -36,6 +36,13 @@ extension Reactive where Base: UITextField {
         )
     }
     
+    /// Bindable sink for `attributedText` property.
+    public var attributedText: UIBindingObserver<Base, NSAttributedString?> {
+        return UIBindingObserver(UIElement: self.base) { textField, attributedText in
+            textField.attributedText = attributedText
```

Comment:
I think we need to find way to make sure this isn't updated in case attributed text is the same and test this.
```
if textField.attributedText != attributedText {
    textField.attributedText = attributedText
}
```

We had issues with IME.

```
// This check is important because setting text value always clears control state
// including marked text selection which is imporant for proper input 
// when IME input method is used.
```